src/styles/components/footer/footer.module.scss

```diff
@@ -0,0 +1,124 @@
+.footer {
+    display: flex;
+    flex-direction: column;
+    background-color: var(--color-maroon);
+}
+
+.body {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 36px 146px;
+    color: var(--color-gold);
+}
+
+.pages {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+
+    font-weight: bold;
+    font-size: 1rem;
+    gap: 22px;
+    list-style: none;
+}
+
+.link {
+    text-decoration: none;
+    color: var(--color-gold);
+}
+
+.contact {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    p {
+        display: flex;
+        justify-content: center;
+        text-align: center;
+        line-height: 1.5;
+    }
+    h3 {
+        display: flex;
+        justify-content: center;
+    }
+
+    width: 25%;
+    font-size: 1rem;
+    gap: 5px;
+
+}
+
+.icons {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.circle_frame {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    border-radius: 50%;
+    background-color: var(--color-white);
+    color: var(--color-maroon);
+    width: 63px;
+    height: 63px;
+    font-size: 2.7rem;
+    
+    text-decoration: none;
+    transition: margin 0.2s ease-in-out;
+}
+
+.circle_frame_facebook {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    
+    border-radius: 50%;
+    background-color: var(--color-white);
+    color: var(--color-maroon);
+    width: 63px;
+    height: 63px;
+    font-size: 3.5rem;
+    
+    text-decoration: none;
+    transition: margin 0.2s ease-in-out;
+}
+
+.circle_frame:hover {
```

Comment:
Same concept as above: we can combine the two `:hover` classes.
Also, very nice touch with adding animations!!